contracts/src/v0.8/llo-feeds/VerifierProxy.sol

```diff
@@ -1,12 +1,12 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity 0.8.16;
 
-import {ConfirmedOwner} from "./ConfirmedOwner.sol";
+import {ConfirmedOwner} from "../ConfirmedOwner.sol";
 import {IVerifierProxy} from "./interfaces/IVerifierProxy.sol";
 import {IVerifier} from "./interfaces/IVerifier.sol";
-import {TypeAndVersionInterface} from "./interfaces/TypeAndVersionInterface.sol";
-import {AccessControllerInterface} from "./interfaces/AccessControllerInterface.sol";
-import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {TypeAndVersionInterface} from "../interfaces/TypeAndVersionInterface.sol";
+import {AccessControllerInterface} from "../interfaces/AccessControllerInterface.sol";
+import {IERC165} from "../vendor/IERC165.sol";
```

Comment:
How would you feel about following semver in the `typeAndVersion` for these contracts moving forward? For example this change can be `VerifierProxy 0.0.2` and the Verifier would be `Verifier 0.0.3`. This may help us when we are deploying various versions of contracts on-chain.